### ISSUE TITLE
List of games with problems

### ISSUE THREAD

**Issue by ghost** [ClusterM/hakchi2#582]
Before anything else: Great job! So awesome that you do this.

Now for the games I found not working in the right way:
Terranigma has some heavy sound glitches which make it practically unenjoyable,
Illusion of Gaia has the same sound glitches and goes to blackscreen after the intro,
Secret of Mana just gives me an Error-messages and won't even start (same for A Link to the past btw. I did install it manually via a rom just to see if it works but I just changed to the version which was already installed so no big deal just interesting).
Finally Tetris/Dr. Mario works but has a different resolution than all other games and just won't work properly with the borders. Wouldn't consider it a bug more of an inconvinience.

**Comment by Jonny0r** [comment]
Hello,

all this problem belong to the Internal emulator of Nintendo, not all games are working (good). Please install RetroArch Plugin and add the command line --retroarch and your missing games will work.

Regards,

**Comment by ghost** [comment]
dont really want to install an emulator. like the thought that the games are being emulated by nintendos own software. maybe there can be fixes for the games or a little mod for nintendos emulation. that would be great. if not i can be fine with my 50 other games which work perfactly :) just wanted to give the information in case it could help someone

**Comment by Jonny0r** [comment]
This issues are already known. There is already a huge list on google if you want to have a look.

https://docs.google.com/spreadsheets/d/12HKfz4ZQBy6Ip5awvh8t2aV5cVswYlnsdKxn9xoIW2Y/edit#gid=2014317767

For Illusion of Gaia there is currently no other work around, but Secret of Mana should be possible to make it running.

**Comment by DecanFrost** [comment]
are you useing --retroarch as global command or per game ?

**Comment by ghost** [comment]
Thanks for showing me the list! Didnt know about it

**Comment by jet1319** [comment]
Act Raiser 2 loads enix screen but goes blank after that  please help

**Comment by gnadenlos** [comment]
Unfortunately Retroarch doesn't help with Secret of Mana 2 and Marvelous - the performance is very bad, so I prefer the lores-fonts and graphics in the official emulator. You also can't use most shaders in Retroarch, because it will cause performance and sound problems. It's better than nothing, but we should try to make the games working with the official emulator.

**Comment by ghost** [comment]
I also think instead of just saying "just use retroarch" the main focus should be to get compatibility as high as possible. We should see retroarch as a work-around not as a solution.

**Comment by 03bgood** [comment]
Wow, this is even worse than the NES Classic. At least that was 90% compatible with all games. I hope this 75% compatibility issue is fixed in the near future.

**Comment by ghost** [comment]
To be honest I think the "75%" translates more to 90%+ if you consider practicality. from the 50 games i added only 5 or so didnt work. i think the not working 25% are mostly obscure titles or games which arent even NTSC.

**Comment by LanceloteHK** [comment]
There are some games on the listing showing "Working" but do not work for me.  I am using super famicom mini.
For those not working games there are two different symptoms. 
One is once the game is selected, the screen go dark and then pop out to the main menu. another one is hold in a black screen and when I press the reset button, it will show a C7 error.

**Comment by dbelyaev** [comment]
@Vogeluff 

> I also think instead of just saying "just use retroarch" the main focus should be to get compatibility as high as possible. We should see retroarch as a work-around not as a solution.

You are talking about internal Nintendo made emulator. This part of software was never intended to be able to run\emulate all the possible games\variations and their mappers. Nintendo tested it and programmed it only with games packaged into console in mind.

On the other hand, there is a retroarch emulator that was made to support wider range of games and which is still in development. That is why people pointed you into this direction by "just saying use retroarch".

So, you need to decide if you want to use original built-in Nintendo emulator which does not support 100% of available games (especially those funky japan RPG ones) or use an retroarch with wider support for some strange and rare mappers (and still not 100% of all games).

**Comment by From90th** [comment]
Was tryin to start "Tetris & Dr. Merio" bundle from different resources (smc and sfc), but the picture is much more larger then firmware frame.
Very sadly that the ActRaiser 2 is too in this List!

**Comment by 03bgood** [comment]
Then how come the NES Classic could run almost any NES game (except for one of the Batman games) without any issues? But SNES Classic has problems with a lot of SNES games? Also, I hate RetroArch! That emulator is such a pain in the ass on almost any system it's ported to. The SNES ROMs probably run worse with RetroArch since games like Kirby's Dream Land 3 had problems being emulated properly on the NES Classic. The game would slow down whenever you entered a room with the foreground being animated.

**Comment by ghost** [comment]
Yes at the moment I have to decide if I want to use retroarch for more compatibility or not but as I said that makes it not a solution. Maybe more games can run on canoe if people work on it.

**Comment by gnadenlos** [comment]
There are two main problems: Enix games crashing after the splash screen and games using High-Res mode. So if we can find a solution for one or both of these problems, the percentage of working games would be much higher.  One solution could fix several games. Because those two problems affect some very good games it's worth it to spend time in researching those problems.

**Comment by 03bgood** [comment]
Wow, this is bad! At least the NES Classic had better  compatibility, but the SNES Classic suffers from that. Way to go, Nintendo! Please fix these issues and don't force us to use RetroArch, if we don't want to.

**Comment by IkoMattorr** [comment]
As I said here: https://github.com/ClusterM/hakchi2/issues/677
Apparently Kirby's Dreamland 3 has saving problems, it only saves when you load a savestate.
I loaded the Japanese version of the game (Hoshi no Kirby 3) and it seems to save correctly, I opened the game several times and my progresses were saved, I never used a savestate and it was canoe BTW.
It still has the transparency problem, that makes the game unplayable during the water sections, I suggest, if someone is interested, to make a temporary work around by just removing the layer completely... I'm unable to do it, I don't have the skills, but I suppose it wouldn't be impossible, by patching the rom, right? Obviously I hope some day the hi-res support will be implemented in some way so that the transparency could be rendered correctly.
